pub-critic/src/app/api/Shared.ts

```diff
@@ -1,5 +1,5 @@
 //export const baseURL = "https://tricky-dove-pullover.cyclic.app";
-export const baseURL = "https://api-pub-critic.fly.dev"; //if the other one is down
+export const baseURL = "https://pub-critic.fly.dev"; //if the other one is down
```

Comment:
Env file?

Reply:
making ti public for now in case it goes wrong, perhaps will make special seperate production deployment